lang/rust/Portfile

```diff
@@ -2,6 +2,11 @@
 
 PortSystem          1.0
 
+# Needed to patch the bootstrap binaries and for the final build
+PortGroup                     legacysupport 1.1
+legacysupport.use_static                    yes
+legacysupport.newest_darwin_requires_legacy 13
+
 name                rust
 version             1.51.0
 revision            1
```

Comment:
Shouldn't Rust's revision be bumped for all these changes?

Reply:
No. Nothing is changed at all for 10.10 and new, and for 10.9 and older rust doesn’t currently build.